src/test/func-test/test_misc_func.cc

```diff
@@ -143,9 +143,14 @@ class TestMiscFunc : public TestMiscFuncSuit {
   CPPUNIT_TEST(test_misc_func_31);
   CPPUNIT_TEST(test_misc_func_32);
   CPPUNIT_TEST(test_misc_func_33);
+#ifdef __linux__
+// windowed mode not supported on non-linux
   CPPUNIT_TEST(test_misc_func_34);
+#endif
   CPPUNIT_TEST(test_misc_func_35);
+#ifdef __linux__
   CPPUNIT_TEST(test_misc_func_36);
```

Comment:
can you add a comment for what this test is that's not supported on non-linux?
Is it also windowed mode?